assignment/scripts/array-practice.js

```diff
@@ -91,4 +98,6 @@ console.log('The animals are now', animalArray);
 //     the favorite foods array with the animals array.
 //     Then log the new array.
 //     It should look something like:
-//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
\ No newline at end of file
+//     ['pizza', 'pasta', 'fish', 'cat', 'bird', 'dog']
+let favoriteArray = [favoriteFoods + animalArray];
```

Comment:
I get why you would think this might work. A `+` combines strings, so why not arrays? Unfortunately, it does not work that way. Instead, JavaScript thinks you are trying to combine strings and _converts your arrays!_

```js
"carbonara,biscotti,mint ice cream" + "fish,cat,bird,dog"
```

Then, once the arrays are converted into strings, they are concatenated.

```js
"carbonara,biscotti,mint ice creamfish,cat,bird,dog"
```

So now you have an array with one weird string in it, instead of an array with seven sensible strings in it. Whoops!